### PR TITLE
[full ci] Treat all errors as warnings during diagnostic.

### LINKED ISSUE #3311: Treat all errors as warnings during diagnostic.

### PR DESCRIPTION
**Pull request by vburenin**
Fixes #3311

### CODE REVIEW

**Comment by hickeng** [comment]
LGTM

**Comment by mdubya66** [comment]
lgtm - please merge after it passes full ci

File: cmd/vic-machine/create/create.go [hunk 1]
```diff
@@ -1350,34 +1348,19 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
...
-	if code > 0 {
-		log.Warningf("%s %s %s", apiTestTxt, vchConfig.Target, diag.UserReadableVCAPITestDescription(code))
+		log.Warningf("Failed to get Virtual Container Host %s. Error: %v", c.Data.DisplayName, err)
 	} else {
-		log.Infof("%s %s %s", apiTestTxt, vchConfig.Target, diag.UserReadableVCAPITestDescription(code))
+		// Checking access to vSphere API
+		if cd, err := executor.CheckAccessToVCAPI(ctx, vch, vchConfig.Target); err == nil {
+			code := int(cd)
+			if code > 0 {
+				log.Warningf("vSphere API Test: %s %s", vchConfig.Target, diag.UserReadableVCAPITestDescription(code))
```
Comment (anchal-agrawal) [inline cmd/vic-machine/create/create.go]: Nit - could we rename `UserReadableVCAPITestDescription` to something terse, such as `DeriveTestMessage` or `DisplayTestMessage`? Also, I believe `log.Warningf` and `log.Warnf` serve the same purpose, and I've only seen the latter used throughout our code.

Comment (vburenin) [inline cmd/vic-machine/create/create.go]: Yah, that makes sense. But not for this PR.

**Pull request merged**